01_01_Basic_Rules.txt

```diff
@@ -788,26 +801,107 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
+an opposed maneuver check. If the target wins, the grapple fails and nothing happens. 
+If the grappler wins, both characters gain the Grappled state, and the attacker 
+starts with control. 
+
+	Characters shooting at a target in the Grappled state may hit both or 
+either. With wide-effect weapons like flamethrowers or buckshot, the attack may 
+be made against either target but the damage is divided evenly among the grappled 
+characters. Attacks with other weapons have a 50% + the shooter's LUC mod to hit 
+the intended target, and otherwise hit the other.
+	
+	While in the Grappled state, most ranged attacks at range R2 or R3 (based 
+on the weapon used) against either character have a 50% chance to hit the other, 
+though shooters may add their LUCK mod to the 50 to hit their chosen target. Wider 
+effect weapons like flamethrowers and buckshot always evenly divide their damage 
+among the grappled characters.
+
+	While in the grappled state, the characters are limited in the actions 
+they can take, can only target each other, and may use only one Action per round. 
+The character with control of the grapple may make Unarmed Strikes, use other 
+combat maneuvers, attempt to Pin, or disengage. The character without control 
+may make Unarmed Strikes, attempt to disengage, or attempt to gain control. 
+Each effort takes one Action.
+
+	While in the Pinned state, a character may only attempt to disengage, and 
+success will return the character to the grappled state, without control, instead 
+of releasing the character from the grapple. The character grappling with a Pinned 
+target may attempt to restrain the target with a further opposed maneuver check and 
+appropriate tool, such as zip cuffs. Success renders the target helpless (until 
+they can break the bonds, which depends on the item used, see item descriptions). 
+Pinned targets are also vulnerable to knockout and kill actions: both render the 
+target unconscious and reduce him or her to 1 health, but a successful kill action 
+also forces the target to make a death saving throw. Knockout and Kill require two 
+consecutively won opposed maneuver checks, and targets additionally add their Shock 
+modifier to their roll when defending against attempts to kill. If a character 
+restrains, knocks out, or kills a grappled target, he or she is still grappled and 
+must spend an Action to disengage.
+	
+	Grappling-specific actions are described below.
+	
+Pin: 	The attacker and defender make an opposed maneuver check. If the attacker 
+	wins, the defender gains the Pinned state.
+Disengage: Characters in control of a grapple may disengage without rolling. 
+	Otherwise both characters choose one of STR-Athletics or DEX-Acrobatics 
+	and roll an opposed check (choices may differ). If the disengager wins, 
+	the grapple ends (and the active character has 3 Actions available this 
+	round).
+Gain Control: The involved characters make an opposed maneuver check as if the 

```

Comment:
If these are supposed to be topics with paragraphs, I believe (from looking at the commit view) that a newline will need to be inserted after the colon.

Reply:
I was hoping those would be read as a definition list. The individual maneuvers are already at topic level so I can't put subsections of grappling on the topic level and there isn't a level underneath it.

In retrospect, I should have put all of the Actions in that list or in the preceding prose instead of dividing them between. I'll log that and the possible style problem as an Issue to keep an eye on (from a clarity perspective).